### PR TITLE
#86 add entity browser component

### LINKED ISSUE #86: Add EntityBrowser component
Add stateful component for grouping `EntityTable` and `SearchBar` components.

## Expected Behaviour

- `EntityBrowser` component, wraps `EntityTable` and `SearchBar` components for displaying and querying data.
-  `entities` prop containing entity data to be displayed/queried.

## Current Behaviour

- `EntityTable` component for displaying entities.
- `SearchBar` component for registering user input and clear/search click events.

## Possible Solution

- Wrap `EntityTable` and `SearchBar` components.
- Maintain table data state and filter based on `onChange/onClick` events.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #86.

Branched off #87 (suggest making your life easier by reviewing after #87 is merged).

## Description

Adds `EntityBrowser` component and data fetching logic.

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [ ] Tests OK.

### CODE REVIEW

File: web/src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import * as React from "react";
+import { ApolloProvider, ApolloClient, InMemoryCache } from "@apollo/client";
 
 import { MainPage } from "./pages";
 
-export const App = () => <MainPage/>;
\ No newline at end of file
+export const App = () => {
+    const client = new ApolloClient({
+        uri: 'http://localhost:4000/graphql',
```
Comment (mark-prins) [inline web/src/App.tsx:8]: another one for the `.env`

File: web/stories/EntityBrowser.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "type": "medicinal_product",
+        }
+    ];
+    const entities = data.map((entityNode: EntityNode) => new Entity(entityNode));
+    return <EntityBrowser entities={entities}/>;
+};
+
+export const withApolloData = () => {
+    const client = new ApolloClient({
+        uri: 'http://localhost:4000/graphql',
```
Comment (mark-prins) [inline web/stories/EntityBrowser.stories.tsx:27]: another configuration item.

**Comment by mark-prins** [comment]
The other thought was to add a message when apollo is down - something to add in another issue

**Comment by katherine-sussol** [comment]
> looking good!
> I fixed the merge conflicts, and then fixed the screw up. Hopefully that was ok?
> @wlthomson want a quick 👀 before we merge it?
> @katherine-sussol are you ok if I merge this in?

Yep! Sounds good to me

**Comment by wlthomson** [comment]
@mark-prins Hard to follow merge diffs, so just merged and resolved conflicts again to be sure. Will go ahead and merge.
